README.md

```
@@ -97,483 +127,409 @@ For template + multi-section documents, pass the same structure through `--optio
 
-> This is a centered blockquote that stands out from the regular text.
+| Feature           | Syntax                 | Notes                                                |
+| ----------------- | ---------------------- | ---------------------------------------------------- |
+| Headings          | `# … #####`            | H1–H5, individually styleable                        |
+| Bold / italic     | `**bold**`, `*italic*` |                                                      |
+| Underline         | `++underline++`        | Custom marker                                        |
+| Strikethrough     | `~~text~~`             | GFM                                                  |
+| Inline code       | ``code``               |                                                      |
+| Code blocks       | ````` fenced           | Optional syntax highlighting per block               |
+| Lists             | `-`, `*`, `1.`         | Bullet, numbered, nested, rich formatting inside     |
+| Tables            | `| a | b |`            | GFM — headers, alignment markers, inline formatting  |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 3; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)